jetty-core/jetty-alpn/jetty-alpn-conscrypt-client/src/test/java/org/eclipse/jetty/alpn/java/client/ConscryptHTTP2ClientTest.java

```diff
@@ -41,7 +41,7 @@
 import static org.junit.jupiter.api.Assertions.assertTrue;
 import static org.junit.jupiter.api.condition.OS.LINUX;
 
-@EnabledOnOs({LINUX}) // TODO review if should be enabled on other OS
+@EnabledOnOs(value = LINUX, architectures = "amd64") // TODO review if should be enabled on other OS
```

Comment:
I would make this annotation the same as the server one.

Say we're on Windows, if there is a problem, the server test will fail anyway, and we may as well fail the client one.

Reply:
You're right, it's time to take care of that TODO.